projects/ad9172/src/main.c

```diff
@@ -291,6 +296,32 @@ int main(void)
 	axi_dac_set_datasel(tx_dac, -1, AXI_DAC_DATA_SEL_DDS);
 #endif /* DAC_DMA_EXAMPLE */
 
+#ifdef IIO_SUPPORT
+	printf("The board accepts libiio clients connections through the serial backend.\n");
+
+	struct axi_dmac_init tx_dmac_init = {
+		"tx_dmac",
+		TX_DMA_BASEADDR,
+		DMA_MEM_TO_DEV,
+		DMA_LAST,
+	};
+	struct axi_dmac *tx_dmac;
+
+	axi_dmac_init(&tx_dmac, &tx_dmac_init);
+
+	struct iio_axi_dac_init_param iio_axi_dac_init_par;
+	iio_axi_dac_init_par = (struct iio_axi_dac_init_param) {
+		.tx_dac = tx_dac,
+		.tx_dmac = tx_dmac,
+		.dac_ddr_base = DDR_MEM_BASEADDR,
+#ifndef PLATFORM_MB
```

Comment:
According to https://github.com/analogdevicesinc/no-OS/commit/b438e7c075dd662925bfbf27123313ba371542c2, this should not be present.

Reply:
You are right. Will change.